### PR TITLE
Fix line index bug and MySQL migration errors in seed_db script

### PR DESCRIPTION
**Pull request by e-wai**
## Notion ticket link

Bug fix

## Implementation description
* Noticed some inconsistency around line indexes. Using mutations/UI for insertion, this should never come up but direct insertion with script commands hides the problem. Think this should solve it
* Made based on https://github.com/uwblueprint/planet-read/pull/39#discussion_r667439779

## Steps to test
1. Run script

## What should reviewers focus on?
* Does this fix the issue?

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters:  `docker exec -it <backend-container-id> /bin/bash -c "black . && isort --profile black ."`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: seed_db.sh [hunk 1]
```diff
@@ -131,93 +131,93 @@ else
...
+        for i in 0 1 2 3 4 5 6 7 8 9
         do
             docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', $i, '');"
         done 
     done 
 
     for STORY_TRANSLATION_ID in 12 
     do 
-        for i in 1 2 3 4 5 6 7 8 9 
+        for i in 0 1 2 3 4 5 6 7 8 9
```
Comment (andrewyguo) [inline seed_db.sh:220]: This leaves a total of 11 lines (10 blank and 1 filled). Let's try to keep the amount of lines consistent to 10 in total. 

`for i in 0 1 2 3 4 5 6 7 8` should be good and also remember to change the line number for the snippet below to 9.

**Review by andrewyguo — CHANGES_REQUESTED** [review]

File: seed_db.sh (hunk 1 above)
Comment (gaoxk) [reply seed_db.sh:220]: story_translation id=12 corresponds to story id = 2. 
`SELECT * FROM story_contents WHERE story_id=5;` results in 10 total lines, indexed 0-9
`SELECT * FROM story_translation_contents WHERE story_translation_id=12;` results in 11 total lines, indexed 0-10 where every line is empty except for line index 10.
@e-wai I think this forloop should run 0-8 due to line 225

**Review by gaoxk — CHANGES_REQUESTED** [review]
see comment on thread with andrew
nit: can you change `ENGLISH_US` -> `ENGLISH_UK` on line 126? It's another unrelated error, thanks

File: seed_db.sh [hunk 2]
```diff
@@ -123,15 +123,15 @@ else
     docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (4, 'ENGLISH_UK', 'TRANSLATE', 4);"
 
     docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'GERMAN', 'TRANSLATE', 2);"
-    docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'ENGLISH_US', 'TRANSLATE', 1);"
+    docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (5, 'ENGLISH_UK', 'TRANSLATE', 1);"
```
Comment (andrewyguo) [inline seed_db.sh:126]: the `user_id` for this entry should be changed to 4 since user 1 doesn't have approval for `ENGLISH_UK`

**Review by andrewyguo — CHANGES_REQUESTED** [review]

**Review by andrewyguo — APPROVED** [review]

**Review by gaoxk — APPROVED** [review]

**Pull request merged**